tests/core/test_serialization.py

```diff
@@ -202,6 +208,14 @@ def test_simple_scene(self):
         assert scenario.sceneToBytes(scene2) == data
         assertSceneEquivalence(scene1, scene2)
 
+    def test_random_scene(self):
```

Comment:
Likewise, `test_random_scene` is misleading; I'd use `test_scene_random_orientation` or something like that.